app/models/manager_refresh/save_collection/saver/sql_helper.rb

```diff
@@ -28,13 +28,19 @@ def build_insert_query(all_attribute_keys, hashes)
               #{values}
         }
 
-        if on_conflict_update
-          insert_query += %{
-            ON CONFLICT (#{unique_index_columns.map { |x| quote_column_name(x) }.join(",")})
-              DO
-                UPDATE
-                  SET #{all_attribute_keys_array.map { |key| build_insert_set_cols(key) }.join(", ")}
-          }
+        if inventory_collection.parallel_safe?
+          if on_conflict == :do_nothing
```

Comment:
This looks like what the `on_conflict_update` method should be doing...can't we change that to
```
def on_conflict_action
  :update
end
```
and override it instead of passing in `:do_nothing` as an arg?  Or get rid of the `on_conflict_update` method and just use the arg.  It is weird to mix both of those.